backend/app/api/localdatasets.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import csv
+from typing import List
```

Comment:
I don't see you using the `List` so probably can remove it, also it's preferable to just use `list` (lowercase), which you don't need to import

Reply:
Removed

Reply:
not addressed. Remove `from typing import List`

Reply:
Changed to type list